src/cz/crcs/ectester/standalone/test/suites/StandaloneWrongSuite.java

```diff
@@ -32,23 +32,23 @@
  * @author David Hofman
  */
 public class StandaloneWrongSuite extends StandaloneTestSuite {
-    private String kpgAlgo;
-    private String kaAlgo;
-    KeyAgreement ka;
-    KeyPairGenerator kpg;
+    private KeyAgreement ka;
+    private KeyPairGenerator kpg;
 
     public StandaloneWrongSuite(TestWriter writer, ECTesterStandalone.Config cfg, TreeCommandLine cli) {
         super(writer, cfg, cli, "wrong", "The wrong curve suite tests whether the library rejects domain parameters which are not curves.",
                 "Supports options:",
                 "\t - gt/kpg-type",
-                "\t - kt/ka-type");
+                "\t - kt/ka-type",
+                "\t - skip (place this option before the library name to skip tests that can potentially cause a freeze)");
     }
 
 
     @Override
     protected void runTests() throws Exception {
-        kpgAlgo = cli.getOptionValue("test.kpg-type");
-        kaAlgo = cli.getOptionValue("test.ka-type");
+        String kpgAlgo = cli.getOptionValue("test.kpg-type");
+        String kaAlgo = cli.getOptionValue("test.ka-type");
+        boolean skip = cli.getArg(1).equalsIgnoreCase("-skip");
```

Comment:
Hmm, this line seems weird, I guess I understand that the `cli` cannot get initialized with the skip option by the suite, because the suite doesn't yet know it will be run when the cli is initialized (sort of a circular issue heh). This probably needs a better solution in the future, like having a static method in the suites that initialized their options into a given `cli` or something of that sort.